docs/sources/administration/configuration.md

```diff
@@ -976,6 +976,40 @@ Sets the minimum interval between rule evaluations. Default value is `1`.
 
 > **Note.** This setting has precedence over each individual rule frequency. If a rule frequency is lower than this value, then this value is enforced.
 
+### max_annotation_age =
+
+Configures for how long alert annotations are stored. Default is 0, which keeps them forever.
```

Comment:
For all max age settings I lack information about what kind of formats you could use to configure it. In docs and in .ini files.

Reply:
:+1: